src/main/webapp/WEB-INF/jsp/scripts.jsp

```diff
@@ -18,10 +18,19 @@
     under the License.
 
 --%>
+
```

Comment:
I found that the console errors were caused by resizing the window. The resize() function was using a call that is not valid for this plugin.

Were there other reasons for this change? I do not see any errors when I removed the erroneous line.